google/cloud/bigquery/dataset.py

```diff
@@ -527,13 +527,33 @@ class Dataset(object):
         "friendly_name": "friendlyName",
         "default_encryption_configuration": "defaultEncryptionConfiguration",
         "storage_billing_model": "storageBillingModel",
+        "max_time_travel_hours": "maxTimeTravelHours",
     }
 
     def __init__(self, dataset_ref) -> None:
         if isinstance(dataset_ref, str):
             dataset_ref = DatasetReference.from_string(dataset_ref)
         self._properties = {"datasetReference": dataset_ref.to_api_repr(), "labels": {}}
 
+    @property
+    def max_time_travel_hours(self):
+        """
+        Optional[int]: Defines the time travel window in hours. The value can be from 48 to 168 hours (2 to 7 days).
```

Comment:
Could you please also add in the comments that value must be an integer expressed in multiples of 24 (48, 72, 96, 120, 144, 168), as [indicated in the documentation](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language)? Thank you!

Reply:
@Linchin  done.